napalm/__init__.py

```diff
@@ -26,6 +27,7 @@ def get_network_driver(vendor):
         'JUNOS': JunOSDriver,
         'JUNIPER': JunOSDriver,
         'FORTIOS': FortiOSDriver,
+#        'IBM': IBMDriver,
```

Comment:
Great! thanks! For the documentation I just need to know where to get the library from (your repo URL) and the minimum OS version, I think what you did already is good enough at least to release now. More documentation will be good but I don't want to block the release : )